optaplanner-core/src/test/java/org/optaplanner/core/impl/score/buildin/simple/SimpleScoreInlinerTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.impl.score.buildin.simple;
+
+import java.util.function.Consumer;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.score.inliner.IntWeightedScoreImpacter;
+import org.optaplanner.core.impl.score.inliner.UndoScoreImpacter;
+
+import static org.junit.Assert.*;
+
+public class SimpleScoreInlinerTest {
+
+    @Test
+    public void buildIntWeightedScoreImpacter() {
```

Comment:
The test is a bit difficult to follow, especially with variables like impacter1, impacter2.

Suggestions:
- splitting the test into two with self-explaining names?
- commenting the main steps?

Reply:
Deferred into https://issues.jboss.org/browse/PLANNER-1584